src/components/index.ts

```diff
@@ -0,0 +1,7 @@
+export { AppButton } from './Button';
+export { AppIcon } from './Icon';
+export { AppLogo } from './Logo';
+export { AppToolbar } from './Toolbar';
+export { AppLayout } from './Layout';
+export { AppExternalLink } from './ExternalLink';
+export { LeftMenu } from './LeftMenu';
```

Comment:
@buckyroberts I really like this. We should implement this for both `components` and `containers`. An added benefit is that we can exclude from this list the components that are used by another component, but shouldn't be used by itself. ie: `LeftSubmenu`